### DISCUSSION TITLE
inert to be supported by all browsers

### DISCUSSION THREAD

**Discussion by salbeira** [decker-edu/decker#37] [Ideas]
Right now we use a polyfill to make inertness work on invisible or inactive elements but according to caniuse the `inert` attribute will be supported by all browsers. In particular firefox will have support in version 112. Right now I have 111 installed and my system tells me I am up to date, so I expect it to become available to the common user very soon.

When should we remove the polyfill from our dependencies?

**Comment by mbotsch** [comment]
Let's remove the polyfill now
